texastribune/salesforce-stripe
# Redirects for domain switch

**NOTE: This branch is intentionally branched off something other than `master`**

#### What's this PR do?
Creates redirects for when this app starts living at `support.texastribune.org`. That means URLs for pages currently in the donations-app -- `levels.html` and `faq.html`, for example -- will need to redirect somewhere meaningful. This takes care of that.

#### Why are we doing this? How does it help us?
So we can eventually delete/stop maintaining donations-app.

#### How should this be manually tested?
+ `make interactive`
+ `yarn run dev`
+ Try the following URLs and confirm they redirect to `/donate`: `/`, `/index.html`, `/faq.html`, `/levels.html`.
+ Confirm `/circle.html` redirects to `/circleform`.

#### Have automated tests been added?
No.

#### Has this been tested on mobile?
NA.

#### Are there performance implications?
None.

#### What are the relevant tickets?
https://3.basecamp.com/3098728/buckets/7046813/todos/1010314871

#### How should this change be communicated to end users?
NA.

#### Next steps?
Configure `checkout.*` redirects.

#### Smells?
NA.

#### TODOs:
NA.

#### Has the relevant documentation/wiki been updated?
NA.

#### Technical debt note
Eventually less.
